use-timescale/backup-restore/index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Backup and restore
+excerpt: Understand how backups, restores, and PITR work in Timescale
+products: [cloud]
+keywords: [backup, restore, pitr, point-in-time recovery]
+tags: [backup, restore, recovery, pitr]
+cloud_ui:
+    path:
+        - [services, :serviceId, operations, management]
+---
+
+# Backup, restore, and PITR
+
+Timescale automatically handles backup and restore for all
+services using the `pgBackRest` tool. You don't need to perform
+backups for your Timescale service manually. 
+
+Timescale also offers self-initiated point-in-time recovery (PITR) via the UI. 
+This allows you to restore your service to any point in the last 3 days in a 
```

Comment:
Dunno if it'd be clearer to say "...allows you to fork a service from any time in the previous 3 days"